app/headerHeight.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { useHeaderHeight } from '@react-navigation/elements';
+import hoistNonReactStatics from 'hoist-non-react-statics';
+
+import { isIOS } from './lib/methods/helpers';
+
+export const headerHeight = isIOS ? 50 : 56;
```

Comment:
Where are you getting this from?
Does it really need to be colocated at the root of the app?

Reply:
I also tried remove this values and use the `headerHeight` from elements, but as it return a different value the dropdown override the server title.

![Screen Shot 2022-06-24 at 00 57 32](https://user-images.githubusercontent.com/14894380/175459493-9505ee64-f153-4c8c-b416-0a4ce724ca6c.png)

This values came from the old `containers/Header` component. I put on this file just to maintain together with the HOC, but as I removed to use the `HeaderHeightContext` from elements, I moved to `app/lib/methods/helpers/navigation/index.ts`.